src/print.jl

```diff
@@ -557,19 +556,21 @@ function constraint_string(print_mode, constraint_object::AbstractConstraint)
     end
 end
 function constraint_string(print_mode, constraint_name,
-                           constraint_object::AbstractConstraint)
+                           constraint_object::AbstractConstraint;
+                           in_math_mode = false)
     constraint_without_name = constraint_string(print_mode, constraint_object)
-    if print_mode == IJuliaMode
+    if print_mode == IJuliaMode && !in_math_mode
         constraint_without_name = _wrap_in_inline_math_mode(constraint_without_name)
     end
-    if isempty(constraint_name)
+    # Names don't print well in LaTeX math mode
```

Comment:
It looks like this changes existing printing behavior. Why didn't any tests break?

Reply:
When printing constraint alone, the constraint name is not in math mode so we add the name (no change here)
When printing the full model in math mode, the whole model is in math mode so we don't print the name (no change here)
When printing the full model in REPL mode, the name has been added and the tests updated